src/elm/Api.elm

```diff
@@ -168,15 +178,17 @@ uploadImage shared file toMsg =
         }
 
 
-communityInvite : Shared -> Symbol -> Eos.Name -> String -> (Result Http.Error () -> msg) -> Cmd msg
-communityInvite shared symbol inviter email toMsg =
+communityInvite : Shared -> Symbol -> Eos.Name -> (Result Http.Error String -> msg) -> Cmd msg
```

Comment:
@lucca65 don't you think we should unify this request via graphql, so that we have one unified way of talking to the backend from the frontend, I can prep a mutation and resolver for this ?

Reply:
why the work? i can’t see what we would gain from it

Reply:
I mean we already have an API that is rest for authentication and invite. we would have extra work and made this task take even longer

please let me know the advantages you can think of